test/integration/konnect_entities_test.go

```diff
@@ -76,10 +76,8 @@ func TestKonnectEntities(t *testing.T) {
 		require.NotEmpty(t, cp.Status.Endpoints)
 		// Example: https://e7b5c7de43.us.cp.konghq.tech - always it will include ".cp.".
```

Comment:
Should we also update the comments to tell that we do not check the `.cp.` section?